workflow-casava/src/main/java/ca/on/oicr/pde/workflows/WorkflowClient.java

```diff
@@ -338,4 +338,11 @@ protected SqwFile createOutputFile(String workingPath, String outputFileName, St
         return file;
     }
 
+    // TODO remove me after sonarcloud does its thing
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)